misp/help.md

```diff
@@ -1,17 +1,15 @@
 # Description
 
-[MISP](http://www.misp-project.org/) is an open source threat sharing platform.
-This plugin utilizes the [MISP API](https://circl.lu/doc/misp/automation/index.html) and leverages the [pymisp](https://github.com/CIRCL/PyMISP) library.
+Open source threat sharing platform.  Gather, store and then find correlations of indicators of compromise
```

Comment:
Keep original link, missing period at end of sentence. This long description should contain more information about the capabilities of the plugin.